string robot_state
float64 joint1
float64 joint2
float64 joint3
float64 height
float64 tray
float64 front_pos_x
float64 front_pos_y
float64 front_pos_z
float64 front_pos_a
int64 table_status_1
int64 table_status_2
int64 table_status_3
int64 table_status_4
int64 table_status_5
int64 table_status_6
int64 table_status_7
int64 table_status_8
int64 table_status_9
int64 table_status_10
int64 table_status_11
int64 table_status_12
int64 table_status_13
int64 table_status_14
int64 table_status_15
int64 table_status_16
float64 Error_Code